README.md

```diff
@@ -13,8 +13,8 @@ You will also need to [set up environment variables](https://www.balena.io/docs/
 
 | Name             | Value  
 | -----------------|------------------------------------                                   
-| CITY_CODE     | The code corresponding to your city, e.g `lisbon,pt` or `london,uk`                     
-| API_KEY | [api key](https://openweathermap.org/appid) retrieved after signing up at `openweathermap.org`|
+| CITY_CODE     | The code corresponding to the latitude and longitude of the desired city                 
```

Comment:
Can we add info about how to get this code and an example of one for Lisbon & London like before?